composer.json

```diff
@@ -38,12 +38,20 @@
         "post-install-cmd": "./vendor/bin/run drupal:site-setup",
         "post-update-cmd": "./vendor/bin/run drupal:site-setup"
     },
-    "repositories": {
-        "drupal": {
+    "repositories": [
+        {
             "type": "composer",
             "url": "https://packages.drupal.org/8"
+        },
+        {
+            "type": "git",
+            "url": "https://github.com/openeuropa/DrupalDriver.git"
```

Comment:
Why is it introduced here?

Reply:
We have to add this everywhere for the "openeuropa/code-review-drupal" dependency.

Reply:
ok